packages/core/content-type-builder/admin/src/utils/toGraphQLName.js

```diff
@@ -0,0 +1,10 @@
+import slugify from '@sindresorhus/slugify';
+
+const toGraphQLName = value =>
```

Comment:
Here you're coupling GraphQL & the CTB logic, not sure it's something we want

Reply:
I agree it would be better to have then detached. However I think it is not so bad to do like this because it ensures that any existing project can install the popular graphql plugin without the application breaking. Wdyt ?

Reply:
I think that installing graphQL and getting an error saying your enum types are not compatible with Graphql would be a valid error as it's a plugin.